app.py

```diff
@@ -170,12 +211,27 @@ def create_club():
     j = request.get_json()
     Session, engine = dbconnect(db_options)
     session = Session()
-    club = Club(
-        name=j.get('name'),
-        school_id=j.get('school_id'),
-        description=j.get('description'),
-        img_type = j.get('img_type')
-    )
+
+    if j.get('img_type') != None:
+        club = Club(
+            name=j.get('name'),
+            school_id=j.get('school_id'),
+            description=j.get('description'),
+            category=j.get('category'),
+            location=j.get('location'),
+            usualTime=j.get('usualTime'),
+            img_type=j.get('img_type')
```

Comment:
instead of doing all this just do j.get(img_type, ''), it will default img_type to empty string if it doesn't exist